### DISCUSSION TITLE
TRP profile question

### DISCUSSION THREAD

**Discussion by dremm-inferniclaw** [Total-RP/Total-RP-3#882] [Q&A] [answered]
Is it possible on dracthyr to have 2 different profiles for dragon form and visage form, and if so, how?

**Comment by Meorawr** [comment, marked as answer]
You can use the Automation feature to swap between profiles automatically based upon dragon and visage forms.

Open the TRP3 window, click "Settings", and then "Automation".

![WowT_irWQeZhOE4](https://github.com/Total-RP/Total-RP-3/assets/287102/57e9874c-5707-4b27-9847-1c9fe32f3605)

**Recommended:** Click the right dropdown and click "Create Profile". This will make the rules that we set up are specific to your currently logged in character.

![WowT_agmR4tah3J](https://github.com/Total-RP/Total-RP-3/assets/287102/2169bedf-29c4-41d9-963c-0cce5acdc464)

Click the left dropdown and select the "Change profile" option near the bottom.

![WowT_4iW5F2wVCS](https://github.com/Total-RP/Total-RP-3/assets/287102/7377ec10-5b5c-4107-8b79-b35aff5b18e2)

In the text field, enter something like the following replacing the profile names with ones that you want to swap between automatically.

```
[altform] Visage Profile Name; Dragon Profile Name
```

![WowT_R236592YVf](https://github.com/Total-RP/Total-RP-3/assets/287102/00c67c4d-5eda-4c8f-9074-cb60465804e5)

**Reply by dremm-inferniclaw** [reply]
Ok, thanks! :)

**Comment by Aisling1013** [comment]
Thank you SO much! I have been trying to figure this out for so long! <3
